### ISSUE TITLE
Use DECdocument only if present in kit building?

### ISSUE THREAD

**Issue by wb8tyw** [endlesssoftware/mmk#99]
Can the kitting procedures be changed to only build and include the DECdocument generated files if DECdocument is present on the building system?

While it still is apparently available from http://www.ttinet.com/decdocument.html, it is not a free product to hobbyists.

**Comment by tesneddon** [comment]
Fair enough. Apologies for letting this hang (as well as other issues). Life has a way of keeping us busy :-)  I will pull together a milestone with a bunch of bits that need fixing and attempt to get a V5.2 release out of the way (a working one...).
